ztag/encoders/protobuf.py

```diff
@@ -52,6 +52,21 @@ def encode(self, zout):
         out.transformed = record.protobuf
 
         out.certificates = []
+        # The chain may be helpful in validating this certificate later on
+        # if some of the parents haven't been previously been by the cert
+        # daemon. Therefore, pass along all raw certificates in the chain.
+        # If chains were guaranteed to be presented in a rasonable order, we
+        # could just pass up [n+1:], but people get this wrong all the time,
+        # so we might as well just pass up the entire chain along with every
+        # certificate, but people get this wrong all the time, so we might
+        # as well just pass up the entire chain along with every certificate.
+        # We will not store this to disk inside of zdb.
+        if len(zout.certificates) > 1:
+            presented_chain = [base64.b64decode(c["raw"]) for c in
```

Comment:
This is dumb. Do we not have the bytes for this already at some point earlier? Or does ztag only see the b64 chain?